### PR TITLE
feat: [CVNG-3974]: Grouped MultiSide Nav

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:
Screenshot 2022-01-18 at 9 32 56 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger eslint

File: src/modules/85-cv/components/MultiItemsSideNav/MultiItemsSideNav.utils.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  groupedCreatedMetrics,
+  setSelectedMetric,
+  onRemoveMetric
+}: OnRemoveGroupAndItemInterface): any => {
+  setCreatedMetrics(oldMetrics => {
+    if (groupedCreatedMetrics) {
+      const copyMetric = Object.values(groupedCreatedMetrics)
+        .map(item => item[0].metricName || '')
+        .reverse()
+        .reverse()
```
Comment (shashidhargg) [inline src/modules/85-cv/components/MultiItemsSideNav/MultiItemsSideNav.utils.ts:26]: Why is this? Reversing 2 times will yield the same result right?

File: src/modules/85-cv/components/MultiItemsSideNav/components/SelectedAppsSideNav/components/GroupedSideNav.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        const [label, items] = groupItem
+        return (
+          <CollapseList key={label}>
+            <CollapseListPanel
+              collapseHeaderProps={{
+                heading: (
+                  <Text
+                    className={cx(css.selectedApp, css.collapseHeading)}
+                    color={Color.GREY_900}
+                    font={{ weight: 'semi-bold', size: 'normal' }}
```
Comment (shashidhargg) [inline src/modules/85-cv/components/MultiItemsSideNav/components/SelectedAppsSideNav/components/GroupedSideNav.tsx:34]: Please use font variation.

File: src/modules/85-cv/pages/health-source/connectors/AppDynamics/Components/AppDMappedMetric/AppDMappedMetric.utils.ts [hunk 3]
```diff
@@ -31,6 +26,8 @@ export function updateSelectedMetricsMap({ updatedMetric, oldMetric, mappedMetri
         serviceInstanceMetricPath: '',
         basePath: BasePathInitValue,
         metricPath: MetricPathInitValue,
+        metricName: updatedMetric,
+        groupName: { label: 'Please Select Group Name', value: 'placeholder' },
```
Comment (shashidhargg) [inline src/modules/85-cv/pages/health-source/connectors/AppDynamics/Components/AppDMappedMetric/AppDMappedMetric.utils.ts:30]: Please move the label to string file.

**Review by shashidhargg — APPROVED** [review]
Added a few comments, please check.

**Review by vasuki-kalluri — APPROVED** [review]

**Pull request merged**